Lib/random.py

```diff
@@ -65,7 +65,7 @@
 
 try:
     # hashlib is pretty heavy to load, try lean internal module first
```

Comment:
This code is technically awkward... It tried to speed up import time and did so by circumventing hashlib which means that it is loading and using the slowest possible sha512 implementation by default (hashlib will pick up openssl 3's accelerated sha512 support by default on most platforms and not use our builtin).  so faster startup time for a slower runtime computation?  thankfully this is only ever used by seed() which is a single/constant number of calls for most programs on tiny data so there is zero reason to care about sha512 performance for its purposes. The slower implementation may still be faster on small seed data anyways due to less setup overhead.

Nothing to do here. This works for random's purposes & thanks for the fixup.  But this being a regression in the first place demonstrates how fragile direct use of internal details can be.  _(and indirectly how much in need of an overhaul hashlib.py could use)_

I'd file an issue rather than leaving this comment in the merged PR void if there were anything concrete to describe and tackle, there isn't. :)